ambry-store/src/main/java/com.github.ambry.store/ScanResults.java

```diff
@@ -0,0 +1,194 @@
+package com.github.ambry.store;
```

Comment:
`./gradlew build && ./gradlew test` was successful except for the following 7 tests which should be unrelated to this PR and should be fixed after #541 is merged.

`com.github.ambry.store.IndexSegmentTest > comprehensiveTest FAILED`

`com.github.ambry.store.IndexTest > findEntriesSinceOnRestartTest[0] FAILED`

`com.github.ambry.store.IndexTest > getIndexSegmentFilesForLogSegmentTest[0] FAILED`

`com.github.ambry.store.IndexTest > findEntriesSinceOnRestartTest[1] FAILED`

`com.github.ambry.store.IndexTest > getIndexSegmentFilesForLogSegmentTest[1] FAILED`

`com.github.ambry.store.IndexTest > recoverySuccessTest[0] FAILED`

`com.github.ambry.server.ServerPlaintextTest > endToEndReplicationWithMultiNodeMultiPartitionMultiDCTest FAILED`